api/src/main/java/uk/gov/dhsc/htbhf/card/controller/CardController.java

```diff
@@ -31,17 +33,20 @@
 
     @PostMapping
     @ApiOperation("Create a new card")
-    @ApiResponses({@ApiResponse(code = 201, message = "Id and description of the new card")})
+    @ApiResponses({@ApiResponse(code = 201, message = "The id of the new card")})
```

Comment:
This is too specific to the contents of the response, I'd describe this as `The response to the create card request`.

Reply:
Updated to be less specific.